src/Filesystem.cpp

```diff
@@ -262,10 +210,10 @@ bool Filesystem::makeDirectory(const std::string& path) const
  */
 bool Filesystem::isDirectory(const std::string& path) const
 {
-	if (!PHYSFS_isInit()) { throw filesystem_not_initialized(); }
+    namespace FS = std::filesystem;
+    if(this->isInit()) { throw filesystem_not_initialized(); }
```

Comment:
This appears to be incorrect -- is the intention to check that the FS is initialized and if not to throw?

Additionally, why the explicit 'this' pointer?

Reply:
Copy-Pasta bug from previous function. 'this' pointer was a fight with initellisense and VA, just forgot to delete it.